pkg/cmd/server/server.go

```diff
@@ -643,7 +651,7 @@ func (s *server) runControllers(config *api.Config) error {
 			s.arkClient.ArkV1(),
 			s.arkClient.ArkV1(),
 			s.sharedInformerFactory.Ark().V1().Schedules(),
-			config.ScheduleSyncPeriod.Duration,
+			gcSyncPeriod,
```

Comment:
this shoulda been `scheduleSyncPeriod` but per previous comment, let's just remove this parameter from the constructor.